### PR TITLE
Implement kpod --rm code

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: cmd/kpod/run.go [hunk 1]
```diff
@@ -136,5 +136,9 @@ func runCmd(c *cli.Context) error {
 		fmt.Printf("%s\n", ctr.ID())
 	}
 	wg.Wait()
+	if createConfig.rm {
+		_ = ctr.Stop()
+		err = runtime.RemoveContainer(ctr, true)
```
Comment (mheon) [inline cmd/kpod/run.go]: Should probably do a "return err" here? Or just return runtime.RemoveContainer?

File: cmd/kpod/run.go [hunk 2]
```diff
@@ -136,5 +136,9 @@ func runCmd(c *cli.Context) error {
 		fmt.Printf("%s\n", ctr.ID())
 	}
 	wg.Wait()
+	if createConfig.rm {
+		_ = ctr.Stop()
```
Comment (mheon) [inline cmd/kpod/run.go]: Don't need this, we're doing a force remove so it will be stopped in Remove if necessary

**Comment by mheon** [comment]
Probably should merge after stop/remove PR is ready

File: cmd/kpod/run.go (hunk 2 above)
Comment (rhatdan) [reply cmd/kpod/run.go]: yes just trying to be nice, but I guess we should rely on the --force

File: cmd/kpod/create.go [hunk 3]
```diff
@@ -348,6 +348,10 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime) (*createConfig, er
 		return nil, errors.Errorf("--pid %q is not valid", c.String("pid"))
 	}
 
+	if c.Bool("detach") && c.Bool("rm") {
+		return nil, errors.Errorf("--rm and -d can not be specified together")
```
Comment (TomSweeneyRedHat) [inline cmd/kpod/create.go]: not a biggy, but I'd suggest --detach instead of disable.

**Comment by baude** [comment]
lgtm

**Comment by rhatdan** [comment]
@rh-atomic-bot r=baude

**Pull request closed without merging**